apps/cms-e2e/src/e2e/app.cy.ts

```diff
@@ -1,13 +1,13 @@
-import { getGreeting } from '../support/app.po';
+import { getTitle } from '../support/app.po';
 
 describe('cms-e2e', () => {
   beforeEach(() => cy.visit('/'));
 
-  it('should display welcome message', () => {
+  it('should have page title', () => {
     // Custom command example, see `../support/commands.ts` file
     cy.login('my-email@something.com', 'myPassword');
 
     // Function helper example, see `../support/app.po.ts` file
-    getGreeting().contains(/Welcome/);
+    getTitle().contains(/cms/);
```

Comment:
⚙️ Cypress: Technically, `contains` is used for querying elements, not for verification. So the correct method would be `should`. But since this test will be removed as soon as we introduce real tests, I am okay with leaving it as is.